lib/Sema/MiscDiagnostics.cpp

```diff
@@ -1328,7 +1328,26 @@ void swift::fixItAvailableAttrRename(TypeChecker &TC,
     return labelStr.empty() ? Identifier() : TC.Context.getIdentifier(labelStr);
   });
 
-  if (auto args = dyn_cast<TupleExpr>(argExpr)) {
+  if (auto args = dyn_cast<TupleShuffleExpr>(argExpr)) {
+      // FIXME: Can we really do this?
+      argExpr = args->getSubExpr();
+  }
+
+  if (auto args = dyn_cast<ParenExpr>(argExpr)) {
```

Comment:
```
@available(*, unavailable, renamed: "newFn1_3(fn:)")
func oldFunc1_3(_ closure: () -> Int) {}

oldFunc1_3 { 0 }
```
In this case, `argsExpr` is
```
(paren_expr implicit type='(() -> Int)' location=... range=... trailing-closure
  (closure_expr ...))
```
Because the current `else` branch doesn't cover `args->hasTrailingClosure()` case,
fix-it tries to add `fn:` label on the closure.
```
test/Sema/availability_renamed.swift:27:1: error: 'oldFunc1_3' has been renamed to 'newFn1_3(fn:)'
oldFunc1_3 { 0 }
^~~~~~~~~~
newFn1_3   fn: 
```

Should we cover this case in `else` clause?